db/seeds/success_stories.rb

```diff
@@ -0,0 +1,50 @@
+def success_stories
+  [
+    {
+      student_name: 'Rob Pando',
+      avatar_path_name: 'rob_pando.jpeg',
+      story_content: 'TOP was a perfect match for my learning style. Not walking me through an entire project but actually forcing me to figure it out on my own, by pointing me in all the right directions in order to complete them. That helped me dig deeper into various important topics, which then gave me the necessary knowledge and confidence to start applying for jobs and landing that dream job. The community was part of it too, first getting help and then helping, it was a great way to learn and make it stick.',
+      job_title: 'Fullstack Developer',
+      social_media_link: 'https://github.com/RobPando'
```

Comment:
I asked each person what they wanted their name to link to, Rob wanted his github 😄